src/components/Daos/Daos.scss

```diff
@@ -66,13 +66,26 @@
 
       h2 {
         color: rgba(104, 155, 214, 1);
-        display: block;
+        display: inline;
```

Comment:
this change will affect other stuff.  Why is it needed?

Reply:
would be safer to use `inline-block` I think.

Reply:
In order to get the envelope to show up next to the H2 title of "Your DAOs", the H2 has to be inline. Note that the H2 is nested in this case so it only effects the H2 element of that class. If you have other suggestions on how to get it to layout properly, I'm all ears.

Reply:
I would use inline-block to get the effect you want while minimizing potential side-effects.  If it looks good visually, I'm OK with that.